packages/core/src/conflict-resolution.ts

```diff
@@ -129,7 +129,12 @@ export async function pickLogToAccept(
 
 export class HistoryLog {
   static fromState(dispatcher: Dispatcher, state: StreamState): HistoryLog {
-    return new HistoryLog(dispatcher, state.log)
+    return new HistoryLog(
+        dispatcher,
+        state.log.map((logEntry) => {
```

Comment:
I don't think this should be necessary anymore now that the state log just holds LogEntry not CommitData

Reply:
I tried that earlier but reverted it because the extra fields kept getting returned making tests fail. I realized today that that was because of a different bug in my code 😞 I've now fixed that bug and removed this change.